CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 All notable changes to this project will be documented in this file.
 
+## 0.42.2
+- adds and uses the FTI-Brand colors
```

Comment:
We have breaking changes, you can go up to 0.43.0 already.
Also try to start with capital letter.

Reply:
changed